src/selectors/exportContext.ts

```diff
@@ -25,8 +25,7 @@ interface Options {
  * @param format
  * @param title     Replace the value of the root thought with a new title.
  */
-export const exportContext = (state: State, context: Context, format: MimeType = 'text/html', { indent = 0, title, excludeSrc, excludeMeta }: Options = {}): string => {
-  const linePrefix = format === 'text/html' ? '<li>' : '- '
+export const exportContext = (state: State, context: Context, format: MimeType = 'text/html', { indent = 0, title, excludeSrc, excludeMeta }: Options = {}, depth = 0): string => {
```

Comment:
Can you add `depth` to the `Options` object itself? I think that's cleaner.

Reply:
Sure.